### PR TITLE
Fix department response

### PR DESCRIPTION
**Pull request by ekaterinaryzhenkova**

### CODE REVIEW

**Review by SolovyovDmitry — APPROVED** [review]

**Review by SolovyovDmitry — APPROVED** [review]

File: src/Models/Broker/Contracts/Department/CategoryInfo.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace DigitalOffice.Models.Broker.Contracts.Department
+{
+  public class CategoryInfo
```
Comment (SolovyovDmitry) [inline src/Models/Broker/Contracts/Department/CategoryInfo.cs:5]: make all models record

File: src/Models/Broker/Contracts/Department/CategoryInfo.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  {
+    public Guid Id { get; }
```
Comment (SolovyovDmitry) [inline src/Models/Broker/Contracts/Department/CategoryInfo.cs:7]: get; init; all props

File: src/Models/Broker/Contracts/Department/DepartmentResponse.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DigitalOffice.Models.Broker.Contracts.Department
+{
+  public class DepartmentResponse
```
Comment (SolovyovDmitry) [inline src/Models/Broker/Contracts/Department/DepartmentResponse.cs:6]: DepatmentInfo

**Review by SolovyovDmitry — CHANGES_REQUESTED** [review]

File: src/Models/Broker/Contracts/Department/DepartmentResponse.cs (hunk 3 above)
Comment (ekaterinaryzhenkova) [reply src/Models/Broker/Contracts/Department/DepartmentResponse.cs]: fixed

File: src/Models/Broker/Contracts/Department/CategoryInfo.cs (hunk 1 above)
Comment (ekaterinaryzhenkova) [reply src/Models/Broker/Contracts/Department/CategoryInfo.cs]: done

**Review by SolovyovDmitry — APPROVED** [review]

**Pull request merged**